src/pipeline.py

```diff
@@ -26,11 +26,11 @@
 )
 
 image_urls = prompts.apply(
-    "retrieve_laion_by_prompt",
+    "retrieve_from_faiss_by_prompt",
     arguments={
-        "num_images": 2,
-        "aesthetic_score": 9,
-        "aesthetic_weight": 0.5,
+        "url_mapping_path":"hf://datasets/fondant-ai/datacomp-small-clip/id_mapping/id_mapping",
```

Comment:
```suggestion
        "url_mapping_path":"hf://datasets/fondant-ai/datacomp-small-clip/id_mapping",
```
I fixed this on the dataset